### PR TITLE
orrandomForest: switched from jbuilder to dune

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/orrandomForest/orrandomForest.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/orrandomForest"
+bug-reports: "https://github.com/UnixJunkie/orrandomForest/issues"
+dev-repo: "git+https://github.com/UnixJunkie/orrandomForest.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+build: [
+  ["R" "CMD" "BATCH" "files/install_randomForest.r"]
+  ["R" "CMD" "BATCH" "files/install_matrix.r"]
+  ["jbuilder" "build" "-p" name "-j" jobs]
```
Comment (kit-ty-kate) [inline packages/orrandomForest/orrandomForest.2.0.0/opam]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```

**Comment by kit-ty-kate** [comment]
Looking at previous PRs for this package (https://github.com/ocaml/opam-repository/pull/12392) it appears to have succeeded to install on mac. However this time it doesn't seem to be the case, do you have any idea what has changed?

**Comment by UnixJunkie** [comment]
I will check on a mac.

**Comment by UnixJunkie** [comment]
superseded by https://github.com/ocaml/opam-repository/pull/15397

**Pull request closed without merging**